### ISSUE TITLE
Duplicate Mediators

### ISSUE THREAD

**Issue by yourpalmark** [strangeioc/strangeioc#71]
I ran into this in the past once in a test scene and forgot about it, but am now unfortunately running into it again in my current project.

Have you ever noticed 2 mediators being instantiated and attached to a view?

We have a GameObject in the scene that we add a View to at runtime, this in turn is attaching 2 mediators. Can't figure out why.

**Comment by green-coder** [comment]
Is this issue still opened or did the fix above solved it? Please update the issue's status if needed, for clarity.

**Comment by fogAndWhisky** [comment]
Sorry, should have dealt with this ages ago. @prankard's fix solves at least the symptom, so I recommend applying his one-line patch. 

That said, I've never been able to reproduce the case to get at the root cause. Clearly others have, so let's fix this. Does anyone have a solid, simple repro I can look at?

**Comment by brandesign** [comment]
Sorry to comment on this old closed issue. We have exactly the same problem, what seems to be resolved with @prankard's fix.
@fogAndWhisky Are there any plans on applying the patch in strange-core? We are using strange-core as a git submodule and in general we would not like to modify external libs.

**Comment by fogAndWhisky** [comment]
@brandesign Yes, we're *really* close to getting v1 out the door. When we do, we'll update strange-core as well. Sorry for the delay.

This issue is essentially the same as #181. I'll leave both open for now, since it'll remind us not to forget updating strange-core!

**Comment by Bunzaga** [comment]
I noticed this only happens to views which are the first ones attached to the GameObject inside the Inspector.  As an example, I have two views: WindowView and DialogWindowView.  If I 'move up' one above the other, the one directly below the TransformComponent (the first one created) is the one which is duplicated.

BTW, yourpalmark fix also worked for me.
